diff --git a/node_modules/minimatch/minimatch.js b/node_modules/minimatch/minimatch.js
index 2e4a058..e7e1a57 100644
--- a/node_modules/minimatch/minimatch.js
+++ b/node_modules/minimatch/minimatch.js
@@ -9,2 +9,3 @@ minimatch.sep = path.sep
 var GLOBSTAR = minimatch.GLOBSTAR = Minimatch.GLOBSTAR = {}
-var expand = require('brace-expansion')
+var braceExpansion = require('brace-expansion')
+var expand = typeof braceExpansion === 'function' ? braceExpansion : braceExpansion.expand
